doc8.ini

```diff
@@ -0,0 +1,4 @@
+[doc8]
+path=/docs/source
+extensions=.rst
+ignore-path=tcms/templates/*,requirements/*
```

Comment:
This file is useless as-is so remove it:

- `path` is not a config file option
-  `extensions` is not a config file option
- `ignore-path` - still doesn't ignore files ourside docs/sources or txt files

Reply:
- ignore-path enabled - 130 files scanned, disabled 140 files scanned
- extensions should be extension
- path will be removed and scan location invoked via cmd

Reply:
This file is not needed when we run `make doc8` so can you explain why do we need it ?